src/main/resources/assets/tconstruct/book/encyclopedia/zh_tw/upgrades/armor/boots/tconstruct_soulspeed.json

```
@@ -0,0 +1,17 @@
+{
+  "modifier_id": "tconstruct:soulspeed",
+  "text": [
+    {
+      "text": "在靈魂沙上的移動速度+40.5%，後續每級移動速度+10.5%，能夠減少靈魂沙帶來的減速效果。"
```

原版 `Soul Sand` 翻譯為 `靈魂砂`，需統一用字。
```suggestion
      "text": "在靈魂砂上的移動速度+40.5%，後續每級移動速度+10.5%，能夠減少靈魂砂帶來的減速效果。"
```